R/ranger/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ranger"
+version = v"0.13.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/imbs-hl/ranger/archive/refs/tags/v$(version).tar.gz", "de60c5ca6ffab1b6cd17c8058c7736f74944841d782707906ba3c68530688916"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd ranger-0.13.1/cpp_version/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIB=1
+make -j$nproc
+make install
+exit
```

Comment:
> Please advise whether any action is needed.

We have the text of the MIT license in `/usr/share/licenses/`
```suggestion
install_license /usr/share/licenses/MIT
```